### PR TITLE
[capture] updated hook from useScreenCapturePermissions to usePermissions

### PR DESCRIPTION
**Pull request by mrakesh0608**
# Why

# How

# Test Plan

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by behenate — COMMENTED** [review]
Thanks for the contribution! Can you also run the 
`et gdad -p expo-screen-capture` command and commit the changes to update the docs `.json` file?

**Review by behenate — APPROVED** [review]
🎉

File: packages/expo-screen-capture/CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 - Removed redundant usage of `EventEmitter` instance. ([#28946](https://github.com/expo/expo/pull/28946) by [@tsapeta](https://github.com/tsapeta))
 
+- Updated hook from useScreenCapturePermissions to usePermissions in example ([#30076](https://github.com/expo/expo/pull/30076) by [@mrakesh0608](https://github.com/mrakesh0608))
```
Comment (Simek) [inline packages/expo-screen-capture/CHANGELOG.md:17]:
```suggestion
- Updated hook from `useScreenCapturePermissions` to `usePermissions` in example ([#30076](https://github.com/expo/expo/pull/30076) by [@mrakesh0608](https://github.com/mrakesh0608))
```

Changelog entry formatting nit.

**Review by Simek — APPROVED** [review]
LGTM, thanks for the correction! 👍

**Review by amandeepmittal — APPROVED** [review]
Thanks for fixing this 👍

**Pull request merged**